tuning/trainercontroller/operations/logcontrol.py

```diff
@@ -0,0 +1,65 @@
+# Third Party
+from transformers import TrainingArguments
+from transformers.utils import logging
+
+# Local
+from .operation import Operation
+
+logger = logging.get_logger(__name__)
+logger.setLevel(level=logging.DEBUG)
+
+VALID_LOG_LEVELS = {
```

Comment:
Is https://github.com/python/cpython/blob/b93386e1c4dbc871148e081ae147f7d1e8d21444/Lib/logging/__init__.py#L126 usable here?